src/main/resources/data/computercraft/lua/rom/apis/rednet.lua

```diff
@@ -28,6 +28,15 @@ local tReceivedMessages = {}
 local tReceivedMessageTimeouts = {}
 local tHostnames = {}
 
+--[[ Patch the bug whenever a computer with an ID grater than 65535 crashes Rednet upon opening.
+-- It modulos the ID to always have an ID lower or equal to 65532
+-- This makes it compatible with the Modem API
+-- Reason to choose 65532 to modulo is because 65533, 65534 an nd 65535 are used respectivel
+]]
+local function idAsChannel(id)
+    return (id or os.getComputerID()) % 65532
```

Comment:
If you wish `65532` to be used as well, you should modulo by `65533` instead, currently the range is `0-65531`

Reply:
So, it was actually correct!
Used 65533 before and somehow started to have doubts about math.